themes/confirmme/html/message.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends ":"~template~":base.html.twig" %}
+{% block stylesheets %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css" />
```

Comment:
This is maybe a bit nitpicking, but shouldn't we use a bootstrap version that is supported today? Eg, bootstrap v5. For such a small theme, that change seems minimal?

Reply:
@jos0405 one for you!